static/js/constants.js

```diff
@@ -222,3 +222,16 @@ export const FACET_OPTION_LABEL_KEYS = {
 }
 
 export const FACET_ORDER = [BOOTCAMP_FACET_KEY, STATUS_FACET_KEY]
+
+export const CS_101 = "CS_101"
+export const CS_102 = "CS_102"
+export const CS_150 = "CS_150"
+export const CS_0 = "CS_0"
+
+export const CS_ERROR_MESSAGES = {
+  [CS_101]:
+    "Missing required fields. Please modify your name/address and try again.",
+  [CS_102]: "Invalid data. Please modify your name/address and try again.",
+  [CS_150]: "Please try again.",
+  [CS_0]:   "Please try again."
```

Comment:
What's `CS_0`? Is that a real error, or a catch-all?  

```suggestion
  [CS_0]:   "Please try again. If this happens more than once, please contact customer support."
```

Reply:
catch all